plugin/plugins/galgame_plugin/static/main.js

```
@@ -6310,6 +6374,41 @@ async function setOcrBackendSelection({ backendSelection = null, captureBackend
+    }
+  } catch (error) {
+    if (!rapidOcrLangQueuedPayload) {
+      setFlash(error instanceof Error ? error.message : String(error), 'error');
+    }
+  } finally {
+    rapidOcrLangRequestPending = false;
+    const nextPayload = rapidOcrLangQueuedPayload;
+    rapidOcrLangQueuedPayload = null;
+    if (nextPayload) {
+      setRapidOcrLang(nextPayload);
+      return;
```

Biome (2.4.14)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)